pywinauto/unittests/test_keyboard.py

```diff
@@ -113,6 +126,11 @@ def receive_text(self):
         received = ' '
         if sys.platform == 'win32':
             received = self.ctrl.TextBlock()
+        elif sys.platform == 'darwin':
+            # Clear clipboard
+            os.system("pbcopy < /dev/null")
+            SendKeys('{cmd}a{cmd}x',pause=0.2)
+            received = macos_functions.read_from_clipboard()

```

Comment:
`read_from_clipboard` can be moved to macOS branch in `clipboard.py` and aligned with naming in this module.